transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringChannelOption.java

```diff
@@ -58,4 +58,10 @@ private IoUringChannelOption() { }
      * Use {@code IOSQE_ASYNC} when submitting {@link IoUringIoOps}.
      */
     public static final ChannelOption<Boolean> IOSQE_ASYNC = valueOf("IOSQE_ASYNC");
+
+    public static final ChannelOption<Boolean> ENABLE_BUFFER_SELECT_READ = valueOf("ENABLE_BUFFER_SELECT_READ");
+
+    public static final ChannelOption<Short> IO_URING_BUFFER_GROUP_ID =
```

Comment:
Do we even need both of them ? Why not just enable it if the group id != -1 or something like this

Reply:
good idea!